app/views/layouts/application.html.erb

```diff
@@ -14,9 +14,9 @@
     <% flash.each do |name, msg| %>
       <%= content_tag :div, msg, :id => "flash_#{name}" %>
     <% end %>
-		
-		<%= yield %>
 
+		<%= yield %>
+    <%= debug(params) if Rails.env.development? %>
```

Comment:
I know we talked about this at some point, but what does this do again? :confused: